vscode-dotnet-runtime-library/src/Utils/CommandExecutor.ts

```diff
@@ -471,29 +449,25 @@ result: ${commandResult.toString()} had no status or signal.`));
                         return '000751'; // Error code 000751 : The command did not report an exit code upon completion. This is never expected
                     }
                 }
-            }
-            else
-            {
-                if(!commandResult.stdout && !commandResult.stderr)
-                {
-                    return '';
-                }
-                else
-                {
-                    if(commandResult.stdout)
-                    {
-                    this.context?.eventStream.post(new CommandExecutionStdOut(`The command ${fullCommandStringForTelemetryOnly} encountered stdout:
+            })();
+
+            return { status: statusCode, stderr: commandResult.stderr?.toString() ?? '', stdout: commandResult.stdout?.toString() ?? ''}
```

Comment:
We could probably trim here and not have to trim everywhere else where its needed, but there are some situations where whitespace matters so I am hesitant to do so.